data_steward/utils/bq.py

```diff
@@ -676,28 +676,38 @@ def build_and_copy_contents(client, src_dataset, dest_dataset):
 
     for table_item in table_list:
         # create empty schemaed tablle with client object
-        schema_list = get_table_schema(table_item.table_id)
+        try:
+            schema_list = get_table_schema(table_item.table_id)
+        except RuntimeError as re:
+            schema_list = None
```

Comment:
I would ask for a logging statement added here so we know what is happening.

```suggestion
            schema_list = None
            LOGGER.warning(f"No schema available for {table_item.table_id}.  This function will "
                           f"copy table contents without specifying a schema.")
```

Reply:
@lrwb-aou Added the log.